code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -171,6 +171,36 @@
 	else
 		icon_state = "[initial(icon_state)]"
 
+//Single shotgun
+/obj/item/gun/ballistic/revolver/singleshotgun
+	name = "Pardner 77 single-shot shotgun"
+	desc = "An old-world New England Pardner shotgun, 2077 production line - right before the nukes."
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	lefthand_file = 'icons/fallout/onmob/weapons/guns_lefthand.dmi'
+	righthand_file = 'icons/fallout/onmob/weapons/guns_righthand.dmi'
+	icon_state = "pardner"
+	item_state = "shotgundouble"
+	icon_prefix = "pardner"
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/pardner
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY
+	fire_delay = 0.5
+	force = 10
+	slowdown = 0.1
+	extra_damage = 10
```

Comment:
do not do this. do not do this. do not do this.
it will apply to buckshot.
it will mog everyone instantly.
do not give shotguns extra pen or ap. worst mistake of my life.